Source/cmake/OptionsHaiku.cmake

```diff
@@ -199,7 +199,7 @@ if (CMAKE_COMPILER_IS_GNUCC AND "${LOWERCASE_CMAKE_HOST_SYSTEM_PROCESSOR}" STREQ
 	string(APPEND CMAKE_CXX_FLAGS " -march=pentium4")
 endif()
 
-SET_AND_EXPOSE_TO_BUILD(USE_TEXTURE_MAPPER OFF)
+SET_AND_EXPOSE_TO_BUILD(USE_TEXTURE_MAPPER ON)
```

Comment:
I'm pretty sure that at least at some point it was not possible to use TEXTURE_MAPPER without also enabling TEXTURE_MAPPER_GL. Was this changed? Is the commit reintroducing TextureMapperImageBuffer still present in the WebKit2 branch? (I think I removed it, but maybe only in the WebKitLegacy branch).